### PR TITLE
Miscellaneous updates

### PR DESCRIPTION
**Pull request by maxsokolovsky**

### CODE REVIEW

File: cmd/env/cancel.go [hunk 1]
```diff
@@ -59,10 +60,12 @@ func cancelEnvironmentByID(id string) error {
 		params["org"] = org
 	}
 
-	body, err := client.Do(http.MethodPost, uri.CreateResourceURI("cancel", "environment", id, "", params), nil)
+	_, err = client.Do(http.MethodPost, uri.CreateResourceURI("cancel", "environment", id, "", params), nil)
```
Comment (dietb) [inline cmd/env/cancel.go:63]: Close the body, even if you're not reading it.

File: cmd/env/rebuild.go [hunk 2]
```diff
@@ -62,10 +62,12 @@ func rebuildEnvironmentByID(id string) error {
 		params["org"] = org
 	}
 
-	body, err := client.Do(http.MethodPost, uri.CreateResourceURI("rebuild", "environment", id, "", params), nil)
+	_, err = client.Do(http.MethodPost, uri.CreateResourceURI("rebuild", "environment", id, "", params), nil)
```
Comment (dietb) [inline cmd/env/rebuild.go:65]: bodyclose

File: cmd/env/restart.go [hunk 3]
```diff
@@ -58,10 +59,12 @@ func restartEnvironmentByID(id string) error {
 		params["org"] = org
 	}
 
-	body, err := client.Do(http.MethodPost, uri.CreateResourceURI("restart", "environment", id, "", params), nil)
+	_, err = client.Do(http.MethodPost, uri.CreateResourceURI("restart", "environment", id, "", params), nil)
```
Comment (dietb) [inline cmd/env/restart.go:62]: bodyclose

File: cmd/env/revive.go [hunk 4]
```diff
@@ -61,10 +61,12 @@ func reviveEnvironmentByID(id string) error {
 		params["org"] = org
 	}
 
-	body, err := client.Do(http.MethodPost, uri.CreateResourceURI("revive", "environment", id, "", params), nil)
+	_, err = client.Do(http.MethodPost, uri.CreateResourceURI("revive", "environment", id, "", params), nil)
```
Comment (dietb) [inline cmd/env/revive.go:64]: bodyclose

File: cmd/env/stop.go [hunk 5]
```diff
@@ -60,10 +60,12 @@ func stopEnvironmentByID(id string) error {
 		params["org"] = org
 	}
 
-	body, err := client.Do(http.MethodPost, uri.CreateResourceURI("stop", "environment", id, "", params), nil)
+	_, err = client.Do(http.MethodPost, uri.CreateResourceURI("stop", "environment", id, "", params), nil)
```
Comment (dietb) [inline cmd/env/stop.go:63]: bodyclose

**Review by dietb — CHANGES_REQUESTED** [review]

File: cmd/env/cancel.go (hunk 1 above)
Comment (maxsokolovsky) [reply cmd/env/cancel.go:63]: In all these cases, the body is closed earlier in the requests package: https://github.com/shipyard/shipyard-cli/blob/6cafb5a7331d4612eb71a5a7ddb9f2ab389fdd83/requests/requests.go#L78.
The body here is already a slice of bytes, and the actual response Body had already been closed.

**Review by dietb — APPROVED** [review]

**Pull request merged**